packages/graphql-hooks/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "graphql-hooks",
+  "version": "3.2.2",
+  "description": "Graphql Hooks",
+  "main": "lib/graphql-hooks.js",
+  "module": "es/graphql-hooks.js",
+  "unpkg": "dist/graphql-hooks.min.js",
+  "scripts": {
+    "test": "jest",
```

Comment:
Is this still used? I see there is a `jest` script in the top-level `package.json` as well as `jest.config.js`.

Reply:
Nice spot, this is left over as all dev tooling was moved to the root. I'll remove.